### PR TITLE
test: Add locals init tests

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: test/unittests/execute_test.cpp [hunk 1]
```diff
@@ -39,9 +39,9 @@ TEST(execute, select)
 {
     /* wat2wasm
     (func (param i64 i64 i32) (result i64)
-      get_local 0
-      get_local 1
-      get_local 2
+      local.get 0
```
Comment (axic) [inline test/unittests/execute_test.cpp:42]: Both of these are valid, are we using this form now consistently?

**Review by axic — APPROVED** [review]

File: test/unittests/execute_test.cpp (hunk 1 above)
Comment (chfast) [reply test/unittests/execute_test.cpp:42]: I'm using `local.xxx` consistently for some time :) Not all instructions have been converted to the newer form.

Comment (chfast) [reply test/unittests/execute_test.cpp:42]: There are still other occurrences of `{set,get}_local` in other files.

Comment (axic) [reply test/unittests/execute_test.cpp:42]: But is this first commit making it uniform, or there are still places using `get_local`?

Comment (chfast) [reply test/unittests/execute_test.cpp:42]: There are other files having `get_local`.

Comment (gumb0) [reply test/unittests/execute_test.cpp:42]: I always forget what is the preference.

**Review by gumb0 — APPROVED** [review]

File: test/unittests/execute_test.cpp (hunk 1 above)
Comment (chfast) [reply test/unittests/execute_test.cpp:42]: The ones with the `.`.

Comment (axic) [reply test/unittests/execute_test.cpp:42]: > There are other files having get_local.

Is it too many to change them here and now?

**Review by gumb0 — APPROVED** [review]

**Pull request merged**